### DISCUSSION TITLE
Option to show Popups ONLY for Logged-in users

### DISCUSSION THREAD

**Discussion by matt70** [elementor/elementor#24661] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

In the advanced rules for a popup, I miss the feature "Show ONLY for logged in users"
This is important for my community site.
I have quite a lot use cases for pop ups, that are not for visitors but registered and logged in users.
For example I can ask for more participation in the community. This does not make sense to show to site visitors, cause they can not participate until they are registered and logged in...
Thanks Matthias

### Describe the solution you'd like.

below the advanced rule: "Hide for logged in users"
I would like to see the rule: "Show only for logged in users"

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by jazir555** [comment]
Wait, this isn't already included by default? What? I'm actually shocked this isn't a trigger condition. @shilo-ey, this wouldn't be hard to add would it?

**Reply by matt70** [reply]
No, it's not there. Before EIementor I was using a free pop up plugin with this conditions. I wonder why this condition is not there. Should be some kind of standard?

**Comment by rblackfox** [comment]
I can't believe this isn't already included by default either. Also being able to conditionally pop popups by user role would also be beneficial.

**Reply by rblackfox** [reply]
the data is there to do it.

**Reply by matt70** [reply]
No it's not.
Just checked it

**Reply by rblackfox** [reply]
I meant the data to have popups display by user role is there and possible.

**Reply by matt70** [reply]
Should be there.
Don't know why Elementor misses this option 🙄

**Comment by matt70** [comment]
Hm, no comment from the Elementor Team after 5 months.
Would be nice to know why this essential option is still missing?
Thanks Matthias

**Comment by Annaick** [comment]
Here I after almost two years, nothing changed
